modal/_container_entrypoint.py

```diff
@@ -52,6 +53,8 @@
 )
 from .running_app import RunningApp
 
+instrument_imports()
```

Comment:
Nit, but we could perhaps env-var-gate this call + import here, instead of always importing it, potentially reducing container startup impact a little

Reply:
Sounds good, done.